osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineArea.cs

```diff
@@ -18,46 +17,49 @@ namespace osu.Game.Screens.Edit.Compose.Components.Timeline
 {
     public partial class TimelineArea : CompositeDrawable
     {
-        public Timeline Timeline;
+        public Timeline Timeline = null!;
 
         private readonly Drawable userContent;
 
-        public TimelineArea(Drawable content = null)
+        public TimelineArea(Drawable? content = null)
         {
             RelativeSizeAxes = Axes.X;
             AutoSizeAxes = Axes.Y;
 
-            userContent = content ?? Drawable.Empty();
+            userContent = content ?? Empty();
         }
 
         [BackgroundDependencyLoader]
-        private void load(OverlayColourProvider colourProvider)
+        private void load(OverlayColourProvider colourProvider, OsuColour colours)
         {
-            Masking = true;
-
             OsuCheckbox waveformCheckbox;
             OsuCheckbox controlPointsCheckbox;
             OsuCheckbox ticksCheckbox;
 
             InternalChildren = new Drawable[]
             {
-                new Box
-                {
-                    RelativeSizeAxes = Axes.Both,
-                    Colour = colourProvider.Background5
-                },
                 new GridContainer
                 {
                     RelativeSizeAxes = Axes.X,
                     AutoSizeAxes = Axes.Y,
+                    RowDimensions = new[]
+                    {
+                        new Dimension(GridSizeMode.AutoSize),
+                    },
+                    ColumnDimensions = new[]
+                    {
+                        new Dimension(GridSizeMode.Absolute, 140),
+                        new Dimension(),
+                        new Dimension(GridSizeMode.Absolute, 35),
+                        new Dimension(GridSizeMode.Absolute, 110),
```

Comment:
Also, if this is adjusted to align with the right toolbox, then I'd also propose aligning the test gameplay button, too.

| before | after |
| :-: | :-: |
| ![1689790263](https://github.com/ppy/osu/assets/20418176/4cf0c0df-3f3a-4930-af50-dde3b3114eb0) | ![1689790320](https://github.com/ppy/osu/assets/20418176/b13908e6-d4b7-4916-bed3-32f656524b93) |

```diff
diff --git a/osu.Game/Screens/Edit/BottomBar.cs b/osu.Game/Screens/Edit/BottomBar.cs
index b8fed4b935..9a32d5d968 100644
--- a/osu.Game/Screens/Edit/BottomBar.cs
+++ b/osu.Game/Screens/Edit/BottomBar.cs
@@ -57,7 +57,7 @@ private void load(OverlayColourProvider colourProvider, Editor editor)
                             new Dimension(GridSizeMode.Absolute, 170),
                             new Dimension(),
                             new Dimension(GridSizeMode.Absolute, 220),
-                            new Dimension(GridSizeMode.Absolute, 120),
+                            new Dimension(GridSizeMode.Absolute, 140),
                         },
                         Content = new[]
                         {

```

Reply:
Was on my list!